AppCenter/AppCenterTests/MSLogDBStorageTests.m

```diff
@@ -1061,13 +1061,20 @@ - (void)testErrorDeletingOldestLog {
 
 - (void)testCreateFromLatestSchema {
 
+  // If
+  [self.storageTestUtil deleteDatabase];
+
   // When
   self.sut = [MSLogDBStorage new];
 
   // Then
   NSString *currentTable =
       [self.sut executeSelectionQuery:[NSString stringWithFormat:@"SELECT sql FROM sqlite_master WHERE name='%@'", kMSLogTableName]][0][0];
   assertThat(currentTable, is(kMSLatestSchema));
+//  NSArray *result =
```

Comment:
Remove commented code... Actually why were we saving the result before?  It doesn't look like it's used

Reply:
This is the test code, I will remove it!

Reply:
removed! :D